<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gmusc Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="LoginWidget" name="loginPage"/>
      <widget class="LibraryWidget" name="libraryPage"/>
      <widget class="SyncWidget" name="syncPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRemote">
    <property name="title">
     <string>&amp;Remote</string>
    </property>
    <addaction name="actionSynchronize"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>&amp;Playback</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRemote"/>
   <addaction name="menuPlayback"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSynchronize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Synchronize</string>
   </property>
   <property name="toolTip">
    <string>Synchronize with remote server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginWidget</class>
   <extends>QWidget</extends>
   <header>loginwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LibraryWidget</class>
   <extends>QWidget</extends>
   <header>librarywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SyncWidget</class>
   <extends>QWidget</extends>
   <header>syncwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
